README.md

```diff
@@ -28,7 +28,11 @@ You can also use `tut` in posts. See `posts/2016-09-30-subtype-typeclasses.md` f
 
 To preview your changes, you have to install [Rake](https://ruby.github.io/rake/) and [Bundler](https://bundler.io/) first.
 To download and set up all necessary dependencies, run
-
+You might need to run install bundler first
+```bash
+$ gem install bundler
```

Comment:
That's what `rake init` does.

Reply:
It didn't do the trick for me, complaining i need bundle 2. I was on bundle 1.x

Reply:
Oh, I misread what you wrote. This is a pending issue (#251); I will address it separately.

Reply:
okay I will remove this for now.